REGRESSION(259197@main): [CMake] [GLIB] Cannot find dependency flite1 when building with JHBuild

dpino opened the pull request:
#### 187e09665fcd8358a98278111390320c6b49e5f2
<pre>
REGRESSION(259197@main): [CMake] [GLIB] Cannot find dependency flite1 when building with JHBuild
<a href="https://bugs.webkit.org/show_bug.cgi?id=251198">https://bugs.webkit.org/show_bug.cgi?id=251198</a>

Reviewed by Adrian Perez de Castro.

* Tools/glib/dependencies/apt: Add dependency &apos;flite1-dev&apos;.
* Tools/glib/dependencies/dnf: Add dependency &apos;flite&apos;.
* Tools/glib/dependencies/pacman: Add dependency &apos;flite-dev&apos;.

Canonical link: <a href="https://commits.webkit.org/259416@main">https://commits.webkit.org/259416@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/fe32f766e07105c3473c3c17aa9ebf9bdc2257df

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/104860 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/13940 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/37755 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/114129 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/174320 "Built successfully and passed tests") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/108768 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/15070 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/4864 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/97187 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/113154 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/110620 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/11644 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [⏳ 🧪 api-mac](https://ews-build.webkit.org/#/builders/API-Tests-macOS-EWS "Waiting to run tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/35/builds/39163 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/93509 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/26266 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/80827 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/7284 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/27627 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/7380 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/84/builds/4219 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/13436 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/47177 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/6495 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/9169 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

aperezdc reviewed (changes requested):
Could you also add `flite` in the `pacman` file and `flite-devel` in the `dnf` file? Thanks!

dpino committed:
REGRESSION(259197@main): [CMake] [GLIB] Cannot find dependency flite1 when building with JHBuild
https://bugs.webkit.org/show_bug.cgi?id=251198

Reviewed by Adrian Perez de Castro.

* Tools/glib/dependencies/apt: Add dependency 'flite1-dev'.
* Tools/glib/dependencies/dnf: Add dependency 'flite'.
* Tools/glib/dependencies/pacman: Add dependency 'flite-dev'.

Canonical link: https://commits.webkit.org/259416@main
